monitoring/storage_linux.go

```diff
@@ -147,15 +147,16 @@ func (c *storageChecker) checkFsType(ctx context.Context, reporter health.Report
 	return nil
 }
 
-// checkHighWatermark checks the disk usage. A failed warning or critical probe
-// will be reported if the usage percentage is above the set thresholds.
-// If the WatermarkWarning percentage is higher than or equal to the
-// WatermarkCritical percentage, then the check will only ever report critical
-// probes.
-func (c *storageChecker) checkHighWatermark(ctx context.Context, reporter health.Reporter) error {
-	if c.WatermarkCritical == 0 {
+// checkDiskUsage checks the disk usage. A warning or critical probe will be
+// reported if the usage percentage is above the set thresholds.
+func (c *storageChecker) checkDiskUsage(ctx context.Context, reporter health.Reporter) error {
+	if c.HighWatermark == 0 {
 		return nil
 	}
+	if c.LowWatermark > c.HighWatermark {
```

Comment:
I think this is redundant as already being validated in `StorageConfig.CheckAndSetDefaults`?